### PR TITLE
[Storage Blob Service] Get/SetBlobContainerAccessPolicy

### PR DESCRIPTION
**Pull request by JinmingHu-MSFT**

### CODE REVIEW

File: sdk/storage/inc/blobs/blob_sas_builder.hpp [hunk 1]
```diff
@@ -281,6 +286,12 @@ namespace Azure { namespace Storage { namespace Blobs {
      */
     void SetPermissions(BlobSasPermissions permissions);
 
+    /**
+     * @brief Clears the permissions for the blob SAS.
+     * 
+     */
+    void ResetPermisions() { Permissions.clear(); }
```
Comment (katmsft) [inline sdk/storage/inc/blobs/blob_sas_builder.hpp:293]: Why do we have this here?

Comment (JinmingHu-MSFT) [reply sdk/storage/inc/blobs/blob_sas_builder.hpp:293]: mainly for test purpose

Comment (katmsft) [reply sdk/storage/inc/blobs/blob_sas_builder.hpp:293]: It's a struct, just clear in test code.

Comment (JinmingHu-MSFT) [reply sdk/storage/inc/blobs/blob_sas_builder.hpp]: removed

**Pull request merged**